### PR TITLE
Add Reference Google Cloud Resources Across Stacks Tutorial Content

### PR DESCRIPTION
**Pull request by toriancrane**
### Proposed changes

- Adds the content for the Google Cloud-specific stack references and outputs tutorials
- Updates the closing paragraph of the Azure stack ref tutorial to replace incorrect AWS references with Azure references.

### CODE REVIEW

**Review by dirien — APPROVED** [review]
LGTM! But I did not run all the code to see if it's working on runtime.

File: assets/css/bundle.css
Comment (thoward) [inline assets/css/bundle.css:1]: See comment on other recent PR: https://github.com/pulumi/docs/pull/13390#discussion_r1867266943

Please remove the commits for these rendered asset files from this branch (not the files).

File: content/tutorials/stack-outputs-refs-azure/index.md [hunk 1]
```diff
@@ -823,9 +823,7 @@ variables:
 
 ## Next steps
 
-In this tutorial, you created a Lambda function that writes a file to an S3 bucket. You also referenced the documentation to create an EventBridge Scheduler that would run the Lambda function on a scheduled basis.
-
-You exported Lambda properties into stack outputs, and referenced those outputs across stacks using stack references.
+In this tutorial, you created an Azure Blob Storage and Container resource. You also referenced the documentation to create a Blob object that would upload a file to the container. You exported resource properties into stack outputs, and referenced those outputs across stacks using stack references.
```
Comment (thoward) [inline content/tutorials/stack-outputs-refs-azure/index.md:826]: Nice catch!

**Review by thoward — CHANGES_REQUESTED** [review]
LGTM (except the asset bundles as noted)

**Review by thoward — APPROVED** [review]

**Pull request merged**